app/components/show/item/details_component.html.erb

```diff
@@ -64,6 +64,24 @@
         </td>
       </tr>
 
+      <% if doi %>
+        <tr>
+          <th class="col-3" scope="row">DOI</th>
+          <td>
+            <%= doi.join(', ') %>
```

Comment:
Actually I removed the join and ensured the solr attribute is a string in our `SolrDocument` config, so it will come back as a string and be less confusing in the view here.